src/studio/src/designer/DataModeling/Converter/Json/Strategy/GeneralJsonSchemaConverter.cs

```diff
@@ -1005,6 +1005,15 @@ private void HandlePropertiesKeyword(XmlSchemaComplexType complexType, XmlSchema
                 SetFixed(subItem, property.Keywords.GetKeyword<ConstKeyword>());
                 SetDefault(subItem, property.Keywords.GetKeyword<DefaultKeyword>());
 
+                if (subItem is XmlSchemaParticle particle)
+                {
+                    var minItemsKeyword = property.Keywords.GetKeyword<MinItemsKeyword>();
+                    if (minItemsKeyword != default)
```

Comment:
No biggie, but must admit I prefer `minItemsKeyword != null`instad of default for easier reading and without having to think about what the default is.

Reply:
Changed according to suggestion.